src/utils/httpRequestParser.ts

```diff
@@ -179,7 +180,10 @@ export class HttpRequestParser implements RequestParser {
                         const inputFilePath = groups[1];
                         const fileAbsolutePath = await resolveRequestBodyPath(inputFilePath);
                         if (fileAbsolutePath) {
-                            combinedStream.append(fs.createReadStream(fileAbsolutePath));
+                            const buffer = await fs.readFile(fileAbsolutePath);
+                            const fileContent = buffer.toString('utf8');
+                            const resolvedContent = await VariableProcessor.processRawRequest(fileContent);
+                            combinedStream.append(resolvedContent);
```

Comment:
If we really want to support this feature, I think we should add a switch(setting) to turn on/off this behavior that parsing the variables in the included body file.

Reply:
overall setting? that seems a bit excessive, why not a request level setting?

Reply:
like instead of `< filename` we could do `<@ filename` to imply it contains variables needing resolving ? What do you think

Reply:
added this with an optional encoding, seems like a nice option, brings in the default with an option to override and only processes variables in files if we signal it that way with the `@`

Reply:
nice suggestion